src/client/opamCLIVersion.ml

```diff
@@ -27,17 +27,16 @@ let of_string s =
   | _ -> failwith "OpamVersion.CLI.of_string"
 
 let current = of_string @@ OpamVersion.(to_string current_nopatch)
+let default = (2,0)
```

Comment:
```suggestion
let default = (fst current, 0)
```
?

Reply:
I tend to agree with this change. Having it directly related permit the automatic update of the default as major version (no more need to handles it). But this automation makes the change invisible, as opposed to a PR announcing the default cli change, in order to let package maintainers and users update their scripts.

Reply:
From dev meeting: this check could be in the release scripts instead (i.e. release checks that the default cli matches major version)